books/RayTracingTheNextWeek.html

```diff
@@ -2256,7 +2256,319 @@
 
 
 
-Rectangles and Lights
+Quadrilaterals
+====================================================================================================
+We've managed to get more than half way through this three-book series using spheres as our only
+geometric primitive. Time to add our second primitive: the quadrilateral.
+
+
+Ray-Plane Intersection
+-----------------------
+Spheres are generally the first ray tracing primitive because their implicit formula makes it so
+easy to solve for ray intersection. Like spheres, planes also have an implicit formula, and
+ray-plane intersection is even _easier_ to solve.
+
+You may already know this implicit formula for a plane:
+
+  $$ Ax + By + Cz + D = 0 $$
+
+That is, a plane is the set of all points $\mathbf{P} = (x,y,z)$ that satisfy the formula above. It
```

Comment:
Where A,B,C,D are just numbers, and x,y,z are the values of vector, That is, a plane is the set of all points $\mathbf{P} = (x,y,z)$ that satisfy the formula above.